src/compiler/scala/tools/nsc/transform/Delambdafy.scala

```diff
@@ -95,14 +95,16 @@ abstract class Delambdafy extends Transform with TypingTransformers with ast.Tre
 
       // no need for adaptation when the implemented sam is of a specialized built-in function type
       val lambdaTarget = if (isSpecialized) target else createBoxingBridgeMethodIfNeeded(fun, target, functionalInterface, sam)
+      val isSerializable = samUserDefined == NoSymbol || samUserDefined.owner.isNonBottomSubClass(definitions.JavaSerializableClass)
```

Comment:
can you remind me why `FLAG_SERIALIZABLE` is only added to scala functions, but not to lambdas implementing user-defined sam types? IUC, the [LMF doc](https://docs.oracle.com/javase/8/docs/api/java/lang/invoke/LambdaMetafactory.html) says that the flag controls whether the instance is serialized as a `SerializedLambda` or not. (if not, ordinary serialization is used?).